ambry-rest/src/main/java/com.github.ambry.rest/RestServer.java

```diff
@@ -280,8 +284,14 @@ public void shutdown() {
       logger.info("Router close took {} ms", elapsedTime);
       restServerMetrics.routerCloseTime.update(elapsedTime);
 
+      clusterMap.close();
```

Comment:
Fair point. Added this to the finally of the try block in which the `clusterMap` is instantiated.